adyenv6core/resources/localization/adyenv6core-locales_en.properties

```diff
@@ -71,7 +78,13 @@ type.paymentinfo.adyenMultibancoAmount.name= Multibanco Amount
 type.paymentinfo.adyenMultibancoDeadline.name= Multibanco Deadline
 type.paymentinfo.adyenMultibancoEntity.name=Multibanco Entity
 type.paymentinfo.adyenMultibancoReference.name=Multibanco Reference
+type.PaymentInfo.adyenPosReceipt.name=POS Receipt
+type.PaymentInfo.adyenTerminalId.name=POS Terminal ID
 
+type.PaymentInfo.encryptedExpiryMonth.name = Encrypted Credit Card Expiry Month
+type.PaymentInfo.encryptedExpiryYear.name = Encrypted Credit Card Expiry Year
+type.PaymentInfo.encryptedSecurityCode.name = Encrypted Credit Card CVC
+type.PaymentInfo.encryptedCardNumber.name = Encrypted Credit Card Number
 
```

Comment:
this is unrelated to the POS payment method right did we miss this for cards ?

Reply:
Yes this is unrelated. It was a small bug and I fixed it because it was small change.